core/sql/cli/SessionDefaults.cpp

```diff
@@ -193,6 +194,8 @@ SessionDefaults::SessionDefaults(CollHeap * heap)
   setEspStopIdleTimeout(60);
   // Default is 1800 (idle ESPs time out in 30 minutes)
   setEspIdleTimeout(30*60);
+  // Default is 1800 (Compiler Idle time out in 30 minutes)
+  setCompilerIdleTimeout(30*60);
```

Comment:
What do we use the compiler for now? SQL compiles are all done in the embedded compiler. I'm guessing DDL and some utilities are the only reasons we use a separate compiler. Since these operations are relatively infrequent, I wonder if we should make the timeout shorter (say, 5 minutes) since the compiler will be idle most the time. It would reduce the process and memory footprint on the system.